gossip/src/cluster_info.rs

```diff
@@ -166,6 +166,7 @@ pub struct ClusterInfo {
     instance: RwLock<NodeInstance>,
     contact_info_path: PathBuf,
     socket_addr_space: SocketAddrSpace,
+    inhibit_snapshot_serving: bool,
```

Comment:
I like 'no_serve_snapshots'

Reply:
or maybe inhibit_snapshot_serving everywhere internally? I'll see what @brooksprumo likes. Just consistent.

Reply:
Sure. I figured I could name the flag and the property different things (I lean towards "inhibit" for this sort of thing, obviously), but happy to fix that up.

Reply:
How about `serve_snapshots`. I'd like to avoid double negatives whenever possible.